packages/client/src/App.js

```diff
@@ -4,9 +4,9 @@ import './App.css';
 import { HomePage } from './containers/HomePage/HomePage.Container';
 import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
 import { ProductView } from './containers/ProductView/ProductView.container';
-import { AboutUs } from './components/PlaceholderPages/AboutUs.component';
-import { ContactUs } from './components/PlaceholderPages/ContactUs.component';
-import { Login } from './components/PlaceholderPages/Login.component';
+import { AboutUs } from './components/PlaceholderPages/AboutUs/AboutUs.component';
```

Comment:
As the Sprint2 is stricter, I will ask for some renamings and structural changes. 🙏🏽 The structural change is probably something I missed in the initial PR:
Those are pages, so they should all be containers, not components. 
They are also not PlaceholderPages in the context of the app, so they should move out of this folder.

Reply:
okay I will move them to container